docs/reference/setup/logging-config.asciidoc

```diff
@@ -194,3 +196,36 @@ files (four rolled logs, and the active log).
 
 You can disable it in the `config/log4j2.properties` file by setting the deprecation
 log level to `error`.
+
+
+[float]
+[[json-logging]]
+=== Json log format
+
+To make parsing elasticsearch logs easier, logs are now printed in a json format.
+This is configured by a log4j layout property `appender.rolling.layout.type = ESJsonLayout`.
+This layout requires a type_name attribute to be set which is used to distinguish
+logs streams when parsing.
+[source,properties]
+--------------------------------------------------
+appender.rolling.layout.type = ESJsonLayout
+appender.rolling.layout.type_name = server
+--------------------------------------------------
+
+Each line contains a single json with set of attributes that are configured in `ESJsonLayout`.
+See this class javadoc for more details.
+When the log event contains a `Throwable`, json will be formatted multiline where first line contains
```

Comment:
I think this too technical. How about: "Exception stack traces will be formatted as multiple lines ..."

Reply:
`However if JSON documents contains exception, stack trace will be formatted as a multiline.
`

Reply:
will reword